build.gradle

```diff
@@ -41,10 +41,7 @@ allprojects {
   apply plugin: 'idea'
   apply plugin: 'eclipse'
   group = 'com.sequenceiq'
-  repositories {
-    mavenLocal()
-    maven { url = "$repoUrl" }
-  }
+
```

Comment:
Wouldn't it be a good idea to keep the mavenLocal() for local developent?

Reply:
yes

Reply:
It was duplicated. Same lines from line 37.